icu4c/source/i18n/dtfmtsym.cpp

```diff
@@ -1688,14 +1688,12 @@ struct CalendarDataSink : public ResourceSink {
             if (value.getType() == URES_STRING) {
                 // We are on a leaf, store the map elements into the stringMap
                 if (i == 0) {
-                    LocalPointer<Hashtable> stringMapPtr(new Hashtable(FALSE, errorCode), errorCode);
-                    stringMap = stringMapPtr.getAlias();
+                    // mapRefs will keep ownership of 'stringMap':
+                    stringMap = mapRefs.create(FALSE, errorCode);
+                    if (stringMap == NULL) { return; }
```

Comment:
Please s/NULL/nullptr/ in new code.

Reply:
That would lead to a mix of NULL & nullptr within the same block of code. I don't think that would be an improvement.